### PR TITLE
Add documentation for hotcue colors

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

File: source/chapters/preferences.rst [hunk 1]
```diff
@@ -240,6 +240,47 @@ You can manually add, relink, and remove Mixxx music directories in
...
+
+  You can also make your own palette by clicking the :guilabel:`Edit` button next
+  to the palette name. This will open a new window which allows you to add, modify,
+  reorder and remove colors in the palette and save it under a new name.
+
+  When editing the hotcue palette, the palette editor also allows assigning a
+  specific color to a hotcue number. This setting honored when
+  :guilabel:`Hotcue default color` is set to `By hotcue number`. If no color in
+  the palette has a hotcue number assigned to it, the color of new hotcues will
+  be mapped to the position of the color in the palette.
```
Comment (ferranpujolcamins) [inline source/chapters/preferences.rst:266]: > [...] the color of new hotcues will be mapped to the position of the color in the palette.

I personally don't find this sentence very clear.

Comment (ferranpujolcamins) [reply source/chapters/preferences.rst:266]: This is explained in the next section. Maybe this sentence here can be removed.

File: source/chapters/preferences.rst [hunk 2]
```diff
@@ -240,6 +240,47 @@ You can manually add, relink, and remove Mixxx music directories in
...
+
+**Default Hotcue Color**
+  It is possible to either either select a single color from the current hotcue palette that will be assigned to all newly created hotcue colors, or select the :guilabel:`By hotcue number` options. The latter will automatically assign a color from the palette to new hotcues based on the hotcue number.
```
Comment (ferranpujolcamins) [inline source/chapters/preferences.rst:269]: Two "either"

**Review by ferranpujolcamins — CHANGES_REQUESTED** [review]

File: source/chapters/preferences.rst (hunk 1 above)
Comment (Holzhaus) [reply source/chapters/preferences.rst]: I improved the explanation and added an example.

File: source/chapters/preferences.rst [hunk 3]
```diff
@@ -256,17 +256,43 @@ Selecting Palettes and Default Colors
...
+Editing Palettes
+----------------
+
+The palette editor allows you add, modify, reorder and remove colors in a color
+palette and save it under a new name. You can also delete custom palettes.
+
+To modify a color, double-click it to open the color picker window.
+
+When editing the hotcue palette, the palette editor also allows assigning a
+specific default color to a hotcue number. This setting honored when
```
Comment (ferranpujolcamins) [inline source/chapters/preferences.rst:278]: There's a missing "is" between "setting" and "honored".

File: source/chapters/preferences.rst [hunk 4]
```diff
@@ -256,17 +256,43 @@ Selecting Palettes and Default Colors
...
+That makes it possible to assign just a few distinct colors to your hotcues
+automatically, but still have more colors available in the palette in case you
+want to override the color manually.
+
+For example, if you have a color palette consisting of 4 colors, and you
+configure the palette to assign the first color to hotcue 1 and the third color
+to hotcue 2, then hotcue 1 will use the first color and hotcue 2 will be
+assigned the third color. The colors will repeat when setting a hotcue with a
+number greater than those used in the palette. In the example above, hotcue 3
+will use the first color and hotcue 2 the second color, and so on.
```
Comment (ferranpujolcamins) [inline source/chapters/preferences.rst:290]: I think you meant "hotcue 3
will use the first color and hotcue **4**"

**Review by ferranpujolcamins — CHANGES_REQUESTED** [review]

**Comment by ferranpujolcamins** [comment]
Thank you for working on the manual!

**Review by ferranpujolcamins — APPROVED** [review]

**Comment by Holzhaus** [comment]
Merge?

**Pull request merged**